### DISCUSSION TITLE
OpenPose poses for ControlNet + other resources

### DISCUSSION THREAD

**Discussion by arenasys** [civitai/civitai#272] [Ideas]
I think a place to share poses will be created eventually, but you guys are probably in the best spot to pull it off well. Also, as more ways are developed to give better control of generations, I think there will be more and more different resources that people want to share besides just models. If you dont know what im talking about, see:  ![](https://cdn.discordapp.com/attachments/1032549038664458260/1075248329581133946/qfsywz.png)

**Comment by WASasquatch** [comment]
OpenPose doesn't read the pose images unless I have blend mode on, which then is just sharing the underlying image you are getting the pose from. If I use the poses on black backgrounds, it doesn't follow pose, and just does whatever, usually for some reason super close-up shot. 

**Works** *(Credit to Tinashe and respective photographer)*
![tinashe-sexy-17960_rendered](https://user-images.githubusercontent.com/1151589/220216795-0513d020-16e9-40fb-aef1-01d7dc6f6dc6.png)


***Doesn't*** **work**
![tinashe-sexy-17960_rendered_rendered](https://user-images.githubusercontent.com/1151589/220216839-6fb74c16-f636-4c74-bf09-5c4c404e6bfa.png)

### Edit:

Turns out I had the preprocessor on, which will process a image, but not also just use a pose. So that was just my mistake. :P
